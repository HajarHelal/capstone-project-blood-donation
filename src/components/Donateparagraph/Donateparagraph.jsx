/* eslint-disable jsx-a11y/label-has-associated-control */
import React from 'react';
import { PropTypes } from 'prop-types';

function Donateparagraph({ text }) {
  return (
    <div className="text-sm sm:text-base md:text-lg  mb-10 leading-relaxed w-10/12">{text}</div>
  );
}

Donateparagraph.propTypes = {
  text: PropTypes.string.isRequired,
};
export default Donateparagraph;
