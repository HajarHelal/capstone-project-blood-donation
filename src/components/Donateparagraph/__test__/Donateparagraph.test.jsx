import React from 'react';
import renderer from 'react-test-renderer';
import Donateparagraph from '../Donateparagraph';

// This is an example of a Snapshot. It will create a new snapshot which all future
// tests will use to compare to.
it('renders correctly', () => {
  const tree = renderer.create(<Donateparagraph text="Donate is awsome" />).toJSON();
  expect(tree).toMatchSnapshot();
});
