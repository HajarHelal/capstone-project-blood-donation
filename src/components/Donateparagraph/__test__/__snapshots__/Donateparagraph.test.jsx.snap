// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="text-sm sm:text-base md:text-lg  mb-10 leading-relaxed w-10/12"
>
  Donate is awsome
</div>
`;
